deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,766 @@
+     2. Configure the storage class - to define the storage system pool name, secret reference, `SpaceEfficiency`, and `fstype`.
+
+    #### 1. Create an array secret
+    Create a secret file as follows `array-secret.yaml` and update the relevant credentials:
+
+    ```
+    kind: Secret
+    apiVersion: v1
+    metadata:
+      name: <NAME>
+      namespace: <NAMESPACE>
+    type: Opaque
+    stringData:
+      management_address: <ADDRESS-1, ADDRESS-2> # Array management addresses
+      username: <USERNAME>                   # Array username
+    data:
+      password: <PASSWORD base64>            # Array password
+    ```
+
+    Apply the secret:
+
+    ```
+    $ kubectl apply -f array-secret.yaml
+    ```
+
+    To create the secret using a command line terminal, use the following command:
+    ```bash
+    kubectl create secret generic <NAME> --from-literal=username=<USER> --fromliteral=password=<PASSWORD> --from-literal=management_address=<ARRAY_MGMT> -n <namespace>
+    ```
+
+    #### 2. Create storage classes
+
+    Create a storage class `demo-storageclass-gold-pvc.yaml` file as follows, with the relevant capabilities, pool and, array secret.
+
+    Use the `SpaceEfficiency` parameters for each storage system. These values are not case sensitive:
+    * IBM FlashSystem A9000 and A9000R
+        * Always includes deduplication and compression.
+        No need to specify during configuration.
+    * IBM Spectrum Virtualize Family
+        * `thin`
+        * `compressed`
+        * `deduplicated`
+    * IBM DS8000 Family
+        * `standard` (default value, if not specified)
+        * `thin`
+
+    ```
+    kind: StorageClass
+    apiVersion: storage.k8s.io/v1
+    metadata:
+      name: gold
+    provisioner: block.csi.ibm.com
+    parameters:
+      SpaceEfficiency: <VALUE>          # Optional: Values applicable for Spectrum Virtualize Family are: thin, compressed, or deduplicated
+      pool: <POOL_NAME>	                # DS8000 Family paramater is pool ID
+
+      csi.storage.k8s.io/provisioner-secret-name: <ARRAY_SECRET>
+      csi.storage.k8s.io/provisioner-secret-namespace: <ARRAY_SECRET_NAMESPACE>
+      csi.storage.k8s.io/controller-publish-secret-name: <ARRAY_SECRET>
+      csi.storage.k8s.io/controller-publish-secret-namespace: <ARRAY_SECRET_NAMESPACE>
+
+      csi.storage.k8s.io/fstype: xfs    # Optional: Values ext4/xfs. The default is ext4.
+      volume_name_prefix: <prefix_name> # Optional: DS8000 Family maximum prefix length is 5 characters. Maximum prefix length for other systems is 20 characters.
+    ```
+
+    #### 3. Apply the storage class:
+
+    ```bash
+    $ kubectl apply -f demo-storageclass-gold-pvc.yaml
+    storageclass.storage.k8s.io/gold created
+    ```
+
+  keywords:
+    - IBM
+    - BlockStorage
+    - CSI
+  version: 1.4.0
+  replaces: ibm-block-csi-operator.v1.3.0
+  maturity: stable
+  maintainers:
+    - name: Guang Jiong Lou
```

Comment:
is he?

Reply:
IMO, ideally, we should find a shared email alias to put here

Reply:
do you want to do it in this scope or to open a ticket?

Reply:
I opened CSI-2144